bin/convert-themes-to-rgb.js

```diff
@@ -1,16 +1,19 @@
 const fs = require("fs");
 const path = require("path");
 const tinycolor = require("tinycolor2");
-const colorsWithAlpha = ["toolbar", "toolbar_field"];
+const colorsWithoutAlpha = ["tab_background_text", "frame", "sidebar"];
 const themesPath = path.join(__dirname, "..", "src", "preset-themes");
 
 const makeTinycolor = colorIn => {
   let { a, s } = colorIn;
-  return tinycolor({
+  let newColor = tinycolor({
     ...colorIn,
-    a: Math.floor(a) / 100.0,
     s: Math.floor(s) / 100.0
   });
+  if (a) {
```

Comment:
`a != undefined` to make sure that if `a: 0` is used, that it continues to be like that.

And please add a unit test for this case.

Reply:
Where is this called from? Should I create a new test for it?

Reply:
Oh this is not called from anywhere, it's a tool. Never mind the unit test for this file.

You should be able to undo the change though, as the `a` property is deleted below depending on conditions.